app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -30,6 +29,10 @@ class Hawkular::MiddlewareManager < ManageIQ::Providers::MiddlewareManager
 
     attr_accessor :client
 
+    def middleware_server_groups
```

Comment:
True!... I barely use `:through` and was basing my knowledge only on ruby guides where only many-to-many relationships are mentioned. Changing it! Thnx!